src/SystemCommands-SourceCodeCommands/SycExtractMethodCommand.class.st

```diff
@@ -70,6 +71,15 @@ SycExtractMethodCommand >> setUpOptionToOverrideExistingMethodDuring: aRefactori
 		ref refactoringWarning: 'Method ', selector printString, ' will override method in ', class name]. 
 ]
 
+{ #category : #execution }
+SycExtractMethodCommand >> setUpOptionToSearchInHierarchyDuring: refactoring [
+
+	refactoring setOption: #searchInAllHierarchy toUse:  [ 
+		UIManager default 
```

Comment:
For this PR is ok, but maybe it is a good idea to extract all calls to UIManager to another object. 
In this way we can handle it in a nicer way. 
I would love to have a better interaction with the user, maybe even explaining the consequences of the decision. Something like the repair window in Iceberg, that shows us what is going to do with one or other option.

Reply:
I'll check how Iceberg do that to have some ideas of how improve user interactions in Refactorings

Reply:
We will have to understand how to express the flow. 
May be here we should have have a call to the driver and a driver configuration 
because this is what the option is doing but this interaction could be move to the driver.